content/projects/oop/person/_index.md

```diff
@@ -24,10 +24,9 @@ title: Person
 Your directory structure should look like this.
 
 ```
```

Comment:
@minat-hub are you sure this (node modules) isn't required for the JS learners? 
Perhaps check with Ngoako or Sheena on this? (But not if you're confident)